packages/grid-export/src/features/grid-copy-to-clipboard/transformers/CellTransformer.ts

```diff
@@ -1,48 +1,62 @@
 import { StandardTableColumnConfig } from "@stenajs-webui/grid";
 import { format } from "date-fns";
 import { CustomCellFormatter } from "../../../common/CellFormatters";
+import { alignmentTransformer } from "./AlignmentTransformer";
 
 export const transformItemToCell = <TItem, TItemValue>(
   item: TItem,
   itemValueResolver: StandardTableColumnConfig<
     TItem,
     TItemValue
   >["itemValueResolver"],
+  justifyContentCell: StandardTableColumnConfig<
+    TItem,
+    TItemValue
+  >["justifyContentCell"],
   itemLabelFormatter:
     | StandardTableColumnConfig<TItem, TItemValue>["itemLabelFormatter"]
     | undefined,
   formatter?: CustomCellFormatter<TItem>
 ): string => {
   if (formatter) {
-    return createCell(formatter(item));
+    return createCell(formatter(item), justifyContentCell);
   }
 
   const value = itemValueResolver(item);
 
   if (itemLabelFormatter) {
     const label = itemLabelFormatter?.(value, item);
-    return createCell(label);
+    return createCell(label, justifyContentCell);
   }
 
   if (typeof value === "number" || typeof value === "string") {
-    return createCell(value);
+    return createCell(value, justifyContentCell);
   }
 
   if (typeof value === "boolean") {
-    return createCell(value ? "Y" : "");
+    return createCell(value ? "Y" : "", justifyContentCell);
   }
 
   if (value instanceof Date) {
-    return createCell(format(value, "yyyy-MM-dd HH:mm"));
+    return createCell(format(value, "yyyy-MM-dd HH:mm"), justifyContentCell);
   }
 
   if (value == null) {
-    return createCell("");
+    return createCell("", justifyContentCell);
   }
 
-  return createCell(String(value));
+  return createCell(String(value), justifyContentCell);
 };
 
-const createCell = (value: string | number): string => {
+const createCell = (
+  value: string | number,
+  justifyContentCell: string | undefined
+): string => {
+  if (justifyContentCell) {
+    console.log("Will render CELL: ", justifyContentCell);
+    return `<td style="${alignmentTransformer(
```

Comment:
Won't alignmentTransformer=undefined give us style="text-align: undefined"?

Reply:
Also, use verbs for wording.
Instead of `alignmentTransformer`, use `transformJustifyContentToTextAlign` or similar. The `AlignTransformer.ts` file should still keep its name.